site/en/docs/webstore/best_listing/index.md

```diff
@@ -140,13 +140,12 @@ Promotional images are your chance to capture users' attention and entice them t
 just use a screenshot; your images should communicate the brand and appear professional. Here are
 specifics about each promotional image:
 
-**Small promo tile: 440x280 pixels** &emsp; Appears on the homepage, category pages, and in search
+Small promo tile: 440x280 pixels
+: Appears on the homepage, category pages, and in search
 results.
 
-**Large promo tile: 920x680 pixels** &emsp; Not currently used. You do not need to provide a large
-promo tile at this time.
-
-**Marquee image: 1400x560 pixels** &emsp; This image is used if your item is chosen for the marquee
+Marquee image: 1400x560 pixels
+: This image is used if your item is chosen for the marquee
```

Comment:
```suggestion
: Used if your item is chosen for the marquee
```

Reply:
Definitions are one of the few places where sentence fragments are permitted in English. What would be the subject of the sentence is already present in the term being defined. That makes this an application of rule II-17: omit needless words.

Reply:
When you run into this, you should also make sure the sentence fragments have similar grammatical structure. That's an application of rule II-19: Express coordinate ideas in similar form.

Reply:
Thanks! 👍  Makes sense :)